packages/manager/modules/exchange/src/information/information.module.js

```diff
@@ -2,13 +2,19 @@ import angular from 'angular';
 import 'angular-translate';
 import '@ovh-ux/ui-kit';
 import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import ovhManagerAdvices from '@ovh-ux/manager-advices';
 
 import routing from './information.routing';
 
 const moduleName = 'ovhManagerExchangeDashboardInformation';
 
 angular
-  .module(moduleName, [ngTranslateAsyncLoader, 'oui', 'pascalprecht.translate'])
+  .module(moduleName, [
+    ngTranslateAsyncLoader,
+    'oui',
+    'pascalprecht.translate',
+    ovhManagerAdvices,
```

Comment:
In some project modules we sort deps modules like:

```suggestion
    ngTranslateAsyncLoader,
    ovhManagerAdvices,
    'oui',
    'pascalprecht.translate',
```

Reply:
@mohammed-zahaf you mean alphabetically (your suggestion is not) ?
It's a good practice but not mandatory.